OvmfPkg/OvmfPkg.ci.yaml

```diff
@@ -11,7 +11,7 @@
 {
     ## options defined .pytool/Plugin/LicenseCheck
     "LicenseCheck": {
-        "IgnoreFiles": []
+        "IgnoreFiles": ["OvmfPkg/Include/IndustryStandard/Xen"]

```

Comment:
It might be useful to add a comment, stating that they are imported from Xen, and that they are MIT licences.
For an example of comment, there's on in "MdePkg/MdePkg.ci.yaml".